README.md

```diff
@@ -433,6 +434,35 @@ against a misbehaving resource. The driver will raise an exception after trying
 to connect with a timeout or after an immediate failure. These driver actions
 will affect the circuit and Semian, which can make future calls fail faster.
 
+A useful way to think about the co-operation between bulkheads and circuit
+breakers is through visualizing a failure scenario graphing capacity as a
+function of time. If an incident strikes that makes the server unresponsive
+with a `20s` timeout on the client and you only have circuit breakers
+enabled--you will lose capacity until all workers have tripped their circuit
+breakers. The slope of this line will depend on the amount of traffic to the now
+unavailable service. If the slope is steep (i.e. high traffic), you'll lose
+capacity quicker. The higher the client driver timeout, the longer you'll lose
+capacity for. In the example below we have the circuit breakers configured to
+open after 3 failures:
+
+![resiliency- circuit breakers](https://cloud.githubusercontent.com/assets/97400/22405538/53229758-e612-11e6-81b2-824f873c3fb7.png)
+
+If we imagine the same scenario but with _only_ bulkheads, configured to have
+tickets for 50% of workers at any given time, we'll see the following
+flat-lining scenario:
+
+![resiliency- bulkheads](https://cloud.githubusercontent.com/assets/97400/22405542/6832a372-e612-11e6-88c4-2452b64b3121.png)
+
+Circuit breakers have the nice property of re-gaining 100% capacity. Bulkheads
+have the desirable property of guaranteeing a minimum capacity. If we do
```

Comment:
Do you mean maximum capacity here? The way I envision bulkheads is that they define the maximum number of users (tickets) for a target resource.